src/components/Calculator.js

```diff
@@ -0,0 +1,52 @@
+export function Calculate() {
+  return (
+    <>
+      <ButtonGroup />
+    </>
+
+  );
+}
+
+function UserInput() {
+  return (
+    <div className="result">
+      0
+    </div>
+  );
+}
```

Comment:
- [ ] For a clean code base and to follow best practice kindly put this in a separate file and import it where it will be needed which I think will be ButtonGroup file (that you should also create) :+1: